routes.js

```diff
@@ -1,86 +1,91 @@
-// const express = require('express');
-// const createMyCustomApi = express.Router();
-// const fs = require('fs');
+const express = require('express');
+const membersAPI = express.Router();
+const fs = require('fs');
 
+let data = require('./members_Array.json');
 
+const saveData = (data) => {
+  fs.writeFileSync('members_Array.json', JSON.stringify(data, null, 2));
+};
 
 
-// // Define routes
-// createMyCustomApi.get('/getAllMembers', (req, res) => {
-//   fs.readFile("./members_Array.json",  (err, data) => {
-//     if(err){
-//       res.status(500).send(err.message)
-//     }
-//     res.status(200).send(data.toString());
-//   });
-// });
+// New function to load data
+const loadData = () => {
+  const rawData = fs.readFileSync('./members_Array.json');
+  return JSON.parse(rawData);
+};
 
 
 
-// createMyCustomApi.post('/addNewMember', (req, res) => {
-//   let data = fs.readFileSync("./members_Array.json"); //diavazoume to array pou exoume hdh
-//   let myObject = JSON.parse(data); // to knaoume js object me parsing
-//   let newData = req.body; // dilwnoume newData to reqbody apo postman
-//     myObject.push(newData); // pusharo sto yparoxn js object to body
-//       console.log(myObject);
-//     let newArray = JSON.stringify(myObject);
-//     fs.writeFile("./members_Array.json", newArray, err => {
-//       if(err) throw err;
-//       console.log(newArray)
-//     });
-//     res.sendStatus(200);
-// });
+// Define routes
 
+membersAPI.get('/getAllMembers', (req, res) => {
+  try {
+    const data = loadData();
+    res.status(200).send(data);
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
 
-// createMyCustomApi.put('/updateExistMember', (req, res) => {
-//   const {memberId, role , firstname, lastname} = req.body
-//   if(!memberId) {
-//     return res.status(400).send('Member ID is required');
-//   }
-//   const memberIndex = data.findIndex(member => member.memberId === memberId);
-//   if (memberId === -1) {
-//     return res.status(404).send("Member not found");
-//   }
-//   const updatedMember = {
-//     memberId,
-//     role : role,
-//     firstname : firstname ,
-//     lastname : lastname
-//   };
 
-//   data[memberIndex] = updatedMember
+membersAPI.post('/addNewMember', (req, res) => {
+  try {
+    let myObject = loadData();
+    let newData = req.body; // required body from postman
 
+    const { memberId, role, firstname, lastname } = newData;
```

Comment:
Please stay consistent across code and for all the variables that do not mutate in that function use only `const`
```suggestion
    const myObject = loadData();
    const newData = req.body; // required body from postman

    const { memberId, role, firstname, lastname } = newData;
```

Reply:
Done!